packages/gtk_doc.rb

```diff
@@ -31,14 +31,17 @@ class Gtk_doc < Package
   depends_on 'docbook_xsl'
 
   def self.build
-    system 'pip3 install six'
-    system 'pip install six'
-    system './autogen.sh --with-xml-catalog=/usr/local/etc/xml/catalog'
-    system './configure --with-xml-catalog=/usr/local/etc/xml/catalog'
+    system "./autogen.sh"
+    system "./configure --with-xml-catalog=/usr/local/etc/xml/catalog --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
     system 'make'
   end
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
   end
+
+  def self.postinstall
+    system "pip3 install --upgrade six --prefix #{CREW_PREFIX}"
+    system "pip2 install --upgrade six --prefix #{CREW_PREFIX}"
```

Comment:
Why do we need both sixes?

Reply:
Python 2 and 3

Reply:
We should only need one six module and it should be in the install section, not postinstall.  I know you didn't make these changes initially but as long as we're "fixing" this package...

Reply:
OK. Python 2.

Reply:
Just a warning, gtk_doc fails to build because of XSL stylesheets. Help?

Reply:
I've run across that problem also.  I just removed the docs with `--disable-gtk-doc` in other packages that had this issue.   Not sure what to do in this case.  Maybe try updating the libxslt package?

Reply:
I have an idea.  Why don't you just remove this package from this PR and we can deal with it in a separate PR?  This doesn't affect the code package AFAICT.

Reply:
Also, the pre-built binaries need to be removed. :)

Reply:
What does AFAICT mean?

Reply:
No. That wouldn't work. I'll just revert gtk_doc, add a package called six that it depends on.

Reply:
AFAICT = as far as I can tell